src/libraries/Common/tests/System/Net/EnterpriseTests/setup/docker-compose.yml

```diff
@@ -60,6 +60,9 @@ services:
     hostname: linuxclient
     domainname: linux.contoso.com
     dns_search: linux.contoso.com
+    privileged: true
+    dns:
+      - 8.8.8.8
```

Comment:
I know this is a backport but I just wanted to make sure that we are ok with using the Google DNS here? cc @GrabYourPitchforks

Reply:
This is there specifically because of https://github.com/dotnet/core-eng/issues/15594
If we can fix the underlying problem and make DNS resolution work in our infrastructure, this could be removed.